### DISCUSSION TITLE
Update/rerun input command every n seconds

### DISCUSSION THREAD

**Discussion by fritzrehde** [junegunn/fzf#2774] [Q&A] [answered]
I would like the input displayed in fzf to update every n seconds (similar to the reload(...) command but automatically run every n seconds). Is this possible somehow? Is there a linux tool to rerun the input command every n seconds that could then be piped into fzf?
Thank you!

**Comment by junegunn** [comment]
You can periodically feed input lines to fzf,

```sh
while true; do
  date
  sleep 1
done | fzf
```

but you can't clear the previous list like `reload` does.

I'm not adding event types that are not directly triggered by the user (in this case we can imagine a "timer" event bound to `reload` action) as I believe that can be quite confusing to the users. (e.g. Input list suddenly disappears/changes in the midst of performing various actions; typing in a query, moving the cursor up and down, selecting multiple entries, etc.).

Related:  #1789 #2019

**Comment by wookayin** [comment, marked as answer]
fzf 0.73.0 added `every(n)` event: https://junegunn.github.io/fzf/releases/0.73.0/#3-timer-driven-bindings-with-everyn
